mkdocs.yml

```
@@ -63,7 +63,11 @@ extra_css:
 extra_javascript:
   - https://unpkg.com/tablesort@5.3.0/dist/tablesort.min.js
   - theme/tablesort.js
-copyright: Copyright &copy; 2007 - <span id="copyyear"></span><script>document.getElementById('copyyear').innerHTML = new Date().getFullYear();</script> VCMI team<br>VCMI is not affiliated with Ubisoft. Buy Heroes III on <a href="https://www.gog.com/en/game/heroes_of_might_and_magic_3_complete_edition">gog.com</a>, we don't support piracy.
+copyright: >
+  Copyright &copy; 2007 - <span id="copyyear"></span><script>document.getElementById('copyyear').innerHTML = new Date().getFullYear();</script> VCMI team
+  <p style="font-size:.42rem !important; line-height: 0.5 !important;">VCMI, this page and the VCMI Team are in no way associated with or supported by Ubisoft Entertainment...</p>
```

Sorry, I meant full version, as it is on our current website:
```suggestion
  <p style="font-size:.42rem !important; line-height: 0.5 !important;">VCMI, this page and the VCMI Team are in no way associated with or supported by Ubisoft Entertainment. Heroes, Might and Magic, Heroes of Might and Magic and their respective logos are registered trademarks of Ubisoft Entertainment in the U.S. and/or other countries. All Rights Reserved. All other trademarks are properties of their respective owners.</p>
```